data/human/outfits.txt

```diff
@@ -394,6 +394,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Old Man's Sword"
+	category "Unique"
+	cost 120000
```

Comment:
Maybe have a larger base price, since it'll automatically get depreciated to 25% of that value upon receiving it?